plugins/web/CMakeLists.txt

```diff
@@ -26,10 +26,18 @@ VASTRegisterPlugin(
 
 find_package(OpenSSL REQUIRED)
 
-if (VAST_ENABLE_SYSTEM_RESTINIO)
+option(VAST_ENABLE_BUNDLED_RESTINIO "Always use the bundled restinio" ON)
+add_feature_info("VAST_ENABLE_BUNDLED_RESTINIO" VAST_ENABLE_BUNDLED_RESTINIO
+                 "always use the bundled restinio.")
+if (NOT VAST_ENABLE_BUNDLED_RESTINIO)
   find_package(fmt REQUIRED)
-  find_package(restinio REQUIRED)
-else ()
+  find_package(restinio)
+endif ()
+if (NOT restinio_FOUND)
+  if (NOT VAST_ENABLE_BUNDLED_RESTINIO)
+    message(
+      STATUS "Cannot find installed restinio; falling back to bundled version")
+  endif ()
```

Comment:
If the user opts out explicitly it should rather fail, should it not?

Reply:
We talked about this in the Huddle; I pushed a change in the most recent commit that makes this a bit better.